alfasim_sdk/types.py

```diff
@@ -16,7 +16,7 @@ class ALFAsimType:
 
 @attr.s(kw_only=True)
 class TracerType(ALFAsimType):
```

Comment:
What is the difference between `TracerType`, `TracerModel` and `TracerModelContainer`?

I am asking because I thought that we should have just `TracerModel` and `TracerModelContainer`.

Reply:
Only the `TracerType` will be used ;)
`container_type` and is internally only, and the user will not be in contact with these values.